pettingzoo/test/seed_test.py

```diff
@@ -1,3 +1,5 @@
+from __future__ import annotations
```

Comment:
Okay this file doesn't use any type hinting at all so I think you shouldn't need to have annotations. I think it may be a problem that env.py didn't have annotations or some other root file didn't have annotations imported, so then when you ran that file it raised an error. You can fix that by importing annotations on all the other files which call that root file, but I think they won't be necessary if you fix the root cause. 

I could be wrong but this is my hunch.

Reply:
That seems correct.